build.cake

```diff
@@ -402,6 +404,90 @@ Task("PackageConsole")
         });
     });
 
+Task("PackageChocolatey")
+	.Description("Creates chocolate packages of the console runner")
+    .IsDependentOn("CreateImage")
+	.Does(() =>
+	{
+		var currentImageDir = IMAGE_DIR + "NUnit-" + packageVersion + "/";
+		
+		EnsureDirectoryExists(PACKAGE_DIR);
+		
+		// Note: Since cake does not yet support a working directory and separate output directory for chocolatey, the following copying and hacks are needed.
+		
+		// List with the addins (addin name, primary dll, additional files)
+		var addins = new Tuple<string, string, string[]>[] {
+			new Tuple<string, string, string[]>(
+				"NUnit.Extension.VSProjectLoader",
+				"vs-project-loader.dll",
+				null
+			),
+			new Tuple<string, string, string[]>(
+				"NUnit.Extension.NUnitProjectLoader",
+				"nunit-project-loader.dll",
+				null
+			),
+			new Tuple<string, string, string[]>(
+				"NUnit.Extension.NUnitV2ResultWriter",
+				"nunit-v2-result-writer.dll",
+				null
+			),
+			new Tuple<string, string, string[]>(
+				"NUnit.Extension.NUnitV2Driver",
+				"nunit.v2.driver.dll",
+				new [] {
+					"nunit.core.dll",
+					"nunit.core.interfaces.dll",
+					"nunit.v2.driver.addins"
+				}
+			),
+			new Tuple<string, string, string[]>(
+				"NUnit.Extension.TeamCityEventListener",
+				"teamcity-event-listener.dll",
+				null
+			)
+		};
+		
+		// Install and copy the addin files
+		var toolsDir = "tools";
+		var nugetInstallSettings = new NuGetInstallSettings { OutputDirectory = toolsDir, ExcludeVersion = true };
+		var addinsDir = System.IO.Path.Combine(currentImageDir, "addins");
+		EnsureDirectoryExists(addinsDir);
+		foreach (var addin in addins) {
+			// Install the extension
+			NuGetInstall(addin.Item1, nugetInstallSettings);
+			var addinToolsPath = System.IO.Path.Combine(toolsDir, addin.Item1, "tools");
+			// Copy primary dll
+			var primaryDllPath = System.IO.Path.Combine(addinToolsPath, addin.Item2);
+			CopyFileToDirectory(primaryDllPath, addinsDir);
+			// Copy additional files
+			if (addin.Item3 != null) {
+				foreach (var additionalItem in addin.Item3) {
+					var additionalItemPath = System.IO.Path.Combine(addinToolsPath, additionalItem);
+					CopyFileToDirectory(additionalItemPath, addinsDir);
+				}
+			}
+			// Write the primary dll to the addins file
+			FileAppendLines(System.IO.Path.Combine(currentImageDir, "nunit.engine.addins"), new[] {
```

Comment:
You probably shouldn't use the same name we use for the addins file that is included with the nuget package but has entirely different content. The names of addins files are not used by the engine at all - it just loads all files of the type.